OpenTrons/opentrons-api
# fix(protocol-designer, components): 7.0 candidate a bug fixes

closes  RQA-1080, RQA-1125, RQA-1102, RQA-1109, RAUT-589, RAUT-590, RQA-1110, RQA-1122, RQA-1100, RQA-1141, RQA-1148, RQA-1151

# Overview

Addresses a handful of bugs

# Test Plan

sandbox: https://sandbox.designer.opentrons.com/pd_bug-fixes-7.0/

On the sandbox build, test that each bug got addressed;
1. add a magnetic module for OT-2 protocol, adding labware on top of it, make sure the correct labware is available and make sure the help link goes to the correct place (RAUT-589, RAUT-590)
2. when editing the pipettes, make sure the arrow icon moves up and down when the dropdown is expanded or not (RQA-1080)
3. make sure you can duplicate labware successfully on a flex and Ot-2 protocol (RQA-1125)
4. add a labware to the deck and give it a verrry long nickname, make sure it truncates after ~3 lines (RQA-1102)
5. add a liquid with a verrry long nickname, make sure it truncates properly both on the liquid tab and when you add the liquid to the labware (RQA-1109, RQA-1110)
6. protocol name should truncate on design tab, but the subtitle to the right of it should not (RQA-1122)
7. removal of the `beta` tag. See Linda's comment in RQA-1100 
8. `touch_tip` option should be disabled in advanced settings in the transfer step when you try to touch tip with a labware that doesn't support it. Hovering over it should show a tooltip explaining why its disabled (RQA-1148)
9. the `moveLabware` step in the timeline should have the correct information shown (should display labware, new location, and if gripper is used or manual intervention is required) RQA-1151
10. when you export a protocol with an unused pipette, the modal copy should be updated (RQA-1141)

# Changelog

- various components in `protocol-designer` directory - adding truncated strings, labware to recommended labware list, fixing a url, duplicate labware bug

# Review requests

see test plan

# Risk assessment

low
